examples/tensorflow-mnist-model-inversion/demo.ipynb

```diff
@@ -4,17 +4,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Tensorflow MNIST MIFace Model Inversion demo"
+    "# MIFace MNIST Model Inversion demo for Dioptra deployment\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    ">⚠️ **Warning:** The attack used in this demo is computationally expensive and will take a very long to complete if run using the CPUs found in a typical personal computer.\n",
-    "> For this reason, it is highly recommended that you run these demos on a CUDA-compatible GPU.\n",
-    "\n",
-    "This notebook contains a demonstration of how to use Dioptra to run experiments that investigate the model inversion attack when launched on a neural network model trained on the MNIST dataset."
+    "This notebook contains an end-to-end demostration for the Dioptra architecture that is deployed on the DGX workstation."
```

Comment:
This notebook looks like it is based off of https://github.com/usnistgov/dioptra/blob/main/examples/tensorflow-mnist-classifier/demo-dgx-lab-deployment.ipynb, please refer to this comment for more details/guidance: https://github.com/usnistgov/dioptra/pull/12#discussion_r717780817.